practice-02/index.html

```diff
@@ -19,52 +19,139 @@
     --------------------->
     <header class="header">
         <div class="container">
-            <div class="header__content">
-                <a href="#" class="logo">
-                    <img src="./img/logo.png" alt="logo">
-
-                </a>
-                <nav class="navbar text--black">
-                    <div class="nav__menu">
-                        <ul class="menu__list">
-                            <li class="list__item">
-                                <a class="item font--small" href="#">Product</a>
+            <nav class="header__content navbar navbar-expand-lg ">
+                <div class="container-fluid ">
```

Comment:
dont need `container-fluid` while we have `container`

Reply:
I tried deleting the class container-fluid but got an error, I fixed it and found it necessary to declare 1 more class in the div to edit, this made me think it would take more lines of code, so I kept the class container-fluid . Please let me know your opinion on this matter